app/jobs/build_submission_bundle_job.rb

```diff
@@ -10,11 +10,17 @@ def perform(submission_id)
     begin
       submission.generate_form_1040_pdf
     rescue
-      submission.transition_to!(:failed, error_code: 'PDF-1040-FAIL')
+      submission.transition_to!(:failed, error_code: 'PDF-1040-FAIL', raw_response: ['Engineers should look in Sentry for a crash'])
```

Comment:
instead of adding to the raw_response here, you could actually pass in the error contents by doing like rescue e  and then raw_response e.message. I'm guessing you considered that and decided to go this route for a particular reason.

Reply:
Yeah - mostly I wanted to avoid there being two places to look into something. Sentry is going to have better information, and if there's any sanitization that we want to do, Sentry can do it, so I prefer directing people to look at Sentry.

Reply:
yeahhh... but also will it be clear in sentry which one was causing the failure? I don't know if sentry will reliably link the id of the failed submission in a way that is exposed in the UI (the job uses submission id, and we only display the irs id in the hub. there's not really a reason that this needs to be the case). If we displayed the "main" raised error, it might help match things up in sentry faster but idk

Reply:
maybe we can add the submission id or something else we can look for that would have been raised in sentry to identify?

Reply:
@bytheway875 Yeah - maybe a nice idea to log the exception class and our ID (not IRS submission ID) - I'll add that right now!